workers/unity/Packages/io.improbable.gdk.movement/Behaviours/ServerMovementDriver.cs

```diff
@@ -61,7 +61,7 @@ private void OnClientUpdate(ClientRequest request)
 
             if (Time.time - lastSpatialPositionTime > spatialPositionUpdateDelta)
             {
-                var positionUpdate = new Position.Update { Coords = positionNoOffset.ToSpatialCoordinates() };
+                var positionUpdate = new Position.Update { Coords = Coordinates.FromUnityVector(positionNoOffset) };
```

Comment:
```suggestion
                var positionUpdate = new Position.Update(Coordinates.FromUnityVector(positionNoOffset));
```

Reply:
I know it's using `{ }` in the entity templates... but everything does there so it doesn't stand out as bad

it's a pet peeve / minor nit / feel free to discard

Reply:
Nah I agree, it bugs me too 😉